sysbuild/Kconfig.pprcore

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SUPPORT_PPRCORE
+	bool
+	default y if SOC_NRF54H20_CPUAPP
```

Comment:
I wonder if we shouldn't think about situation where someone might want to use ppr with rad core?

Reply:
discussed on Teams